client/src/components/SiteSearch/SiteSearch.jsx

```diff
@@ -42,26 +45,29 @@ export const SiteSearchComponent = ({ filterPriorityList }) => {
   }, [completed, results, location.search]);
 
   return (
-    <div style={{ display: 'flex', flexDirection: 'row' }}>
-      <div className="col-md-2">
-        <SiteSearchSidebar
-          queryString={query_string}
-          schemes={Object.keys(results).filter((key) => results[key].include)}
-          authenticated={Boolean(user)}
-          searching={loading}
-          results={results}
-        />
-      </div>
-      <div className="col-md-8" style={{ display: 'flex' }}>
-        <SiteSearchListing
-          loading={loading}
-          error={error}
-          filter={filter || 'cms'}
-          results={results[filter] || results[filterPriorityList[0]]}
-        />
-      </div>
-      <div className="col-md-2" />
-    </div>
+    <Section
+      contentClassName="has-common-sidebar"
```

Comment:
**Leftover Code?** I see four instances of `contentClassName="has-common-sidebar"` now, but I think that class is no longer used.

Reply:
Good catch! I will clean those up